tests/apollo/test_skvbc_control_commands.py

```diff
@@ -0,0 +1,155 @@
+# Concord
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+import os.path
+import random
+import unittest
+from os import environ
+
+import trio
+
+from util import blinking_replica
+from util import skvbc as kvbc
+from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX
+
+
+def start_replica_cmd(builddir, replica_id):
+    """
+    Return a command that starts an skvbc replica when passed to
+    subprocess.Popen.
+
+    Note each arguments is an element in a list.
+    """
+    statusTimerMilli = "500"
+    viewChangeTimeoutMilli = "10000"
+    path = os.path.join(builddir, "tests", "simpleKVBC", "TesterReplica", "skvbc_replica")
+    return [path,
+            "-k", KEY_FILE_PREFIX,
+            "-i", str(replica_id),
+            "-s", statusTimerMilli,
+            "-v", viewChangeTimeoutMilli,
+            "-p" if os.environ.get('BUILD_ROCKSDB_STORAGE', "").lower()
+                    in set(["true", "on"])
+            else "",
+            "-t", os.environ.get('STORAGE_TYPE')]
+
+
+class SkvbcControlCommandsTest(unittest.TestCase):
+    """
+        Sends a wedge command and check that the system stops from processing new requests.
+        Note that in this test we assume no failures and synchronized network.
+        The test does the following:
+        1. A client sends a wedge command
+        2. The client verify that the system reached to a super stable checkpoint
+        3. The client tries to initiate a new write bft command and fails
+    """
+
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 4 and c == 0)
```

Comment:
If you want to run this only for a single configuration, let's make it `lambda n, f, c: n == 7`, as it's closer to the real environment.
Actually, why don't you want to run the test for all supported configs?

Reply:
Fixed. Run for all configurations